src/EditorFeatures/CSharpTest/AddAnonymousTypeMemberName/AddAnonymousTypeMemberNameTests.cs

```diff
@@ -86,7 +86,7 @@ class C
 {
     void M()
     {
-        var v = new { P = new { Type = this.GetType(), V = this.ToString() } };
+        var v = new { Value = new { Type = this.GetType(), String = this.ToString() } };
```

Comment:
not in love with this.

Reply:
althought it seems like the original code is in error, if i'm not mistaken?

Reply:
As for me, first change makes naming better. Second maaaaaybe a bit worse, but I verified, that it is valid C# code. Yes, it is almost a keyword, but "V" name is not very descriptive, so I am not sure, If reverting this worth the effort...